src/components/menu-bar/menu-bar.css

```diff
@@ -32,7 +32,7 @@
     font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
     font-size: $menu-bar-standard-font-size;
     font-weight: bold;
-    background-color: $looks-secondary;
```

Comment:
There's actually a few places where the interface is now purple where it used to be blue, like:

<img width="100" alt="Screenshot 2024-05-21 at 3 12 37 PM" src="https://github.com/mitmedialab/prg-raise-playground-scratch-gui/assets/95306112/58917fb7-a59b-4361-8385-6ea270d2175a">

I think it might make more sense to try changing the `looks-secondary` css variable to the value of `motion-primary` variable and assess if that returns us to our preferred blue state ™️